lists/ug/204cbo

```diff
@@ -0,0 +1,50 @@
+{
+  "name": {
+    "en": "Jinja District Non-Governmental Monitoring Committee",
+    "local": ""
+  },
+  "url": "https://www.ngobureau.go.ug/en/services/community-based-organisations",
+  "description": {
+    "en": "Community Based Organisations (CBOs) operating in Jinja District in Uganda are required to register at the district level by the District NGO Monitoring Committee (DNMC) [1]. DNMC will issue a certificate of registration and a permit to the CBO. The certificate of registration will be issued once, while the permit is renewed annually. The certificate of registration contains a unique registration number that belongs to the CBO. There is no publicly available site to verify the ownership and validity of this number.\n\n[1] https://www.ngobureau.go.ug/en/services/community-based-organisations"
+  },
+  "coverage": [
+    "UG"
+  ],
+  "subnationalCoverage": [
+    "UG-E"
+  ],
+  "structure": [
+    "community based organisation"
+  ],
+  "sector": [],
+  "code": "UG-204CBO",
+  "confirmed": true,
+  "deprecated": false,
+  "listType": "secondary",
```

Comment:
I think listType might be 'primary'. If CBOs can't operate without being registered then this is definitely a primary list.